Release version 0.2.0 (#58)
pkgcore/pkgdev: NEWS.rst

Release Notes
=============

pkgdev 0.2.0 (2022-04-10)
-------------------------

> - pkgdev commit: Mangle copyright single year into year range when appropriate

Reviewer: ```suggestion
- pkgdev commit: Mangle copyright header year from single year into year range when appropriate
```

>   when (thanks to Thomas Bracht Laumann Jespersen, #49)

Reviewer: last word should be appropriate, not double when

Author: Good catch!

> - pkgdev commit: Mangle KEYWORDS to be always sorted (Arthur Zamarin, #47)

Reviewer: ```suggestion
- pkgdev commit: Always sort KEYWORDS via mangler (Arthur Zamarin, #47)
```

> - pkgdev commit: For new packages, the title will include version added, with

Reviewer: ```suggestion
- pkgdev commit: For new packages, include version in commit message ("new package, add ${PV}")
```
